_includes/v2.1/zone-configs/create-a-replication-zone-for-a-database.md

```diff
@@ -0,0 +1,28 @@
+To control replication for a specific database, use the `CONFIGURE ZONE` statement to define the values you want to change (other values will not be affected):
```

Comment:
I would make this mention of the statement a link unless there's a specific reason you'd rather not.  Same comment re: all the other places `CONFIGURE ZONE` is used.